lib/addons/aws-privateca-issuer/index.ts

```diff
@@ -0,0 +1,95 @@
+// lib/kubevious_addon.ts
+import { Construct } from 'constructs';
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import merge from "ts-deepmerge";
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { dependable } from '../../utils';
+import { setPath } from '../../utils/object-utils';
+import { ClusterInfo, Values } from "../../spi";
+import { createNamespace } from "../../utils";
+/**
+ * User provided options for the Helm Chart
+ */
+export interface AWSPrivateCAIssuerAddonProps extends HelmAddOnUserProps {

```

Comment:
Are there any configuration options from the helm chart that would make sense to promote to the root level?

Reply:
These are the available values
https://github.com/cert-manager/aws-privateca-issuer/blob/main/charts/aws-pca-issuer/values.yaml
I couldnt find anything which required to be promoted to root level